meta-networking/recipes-connectivity/autossh/autossh_1.4g.bb

```diff
@@ -0,0 +1,30 @@
+DESCRIPTION = "autossh is a program to start a copy of ssh and monitor it, restarting it as necessary should it die or stop passing traffic"
+HOMEPAGE = "https://www.harding.motd.ca/autossh/"
+
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://autossh.spec;md5=80a6701134723fd3420e733b46a0eb97"
```

Comment:
is this file the right one to checksum ? I wonder if it should check for header files or source files which have copyright headers.

Reply:
The license type is specified on line five of autossh.spec, so I decided that file would be best to checksum.  I'm open to other ideas.